Gemfile.lock

```diff
@@ -1,135 +1,252 @@
       ffi (>= 1.3.0)
-    execjs (2.6.0)
-    faraday (0.9.2)
+    eventmachine (1.2.5)
+    execjs (2.7.0)
+    faraday (0.15.0)
       multipart-post (>= 1.2, < 3)
-    ffi (1.9.10)
-    ffi (1.9.10-x64-mingw32)
-    gemoji (2.1.0)
-    github-pages (44)
-      RedCloth (= 4.2.9)
-      github-pages-health-check (= 0.6.0)
-      jekyll (= 3.0.2)
-      jekyll-coffeescript (= 1.0.1)
-      jekyll-feed (= 0.3.1)
-      jekyll-gist (= 1.4.0)
-      jekyll-mentions (= 1.0.0)
+    ffi (1.9.23)
+    forwardable-extended (2.6.0)
+    gemoji (3.0.0)
+    github-pages (182)
+      activesupport (= 4.2.9)
+      github-pages-health-check (= 1.4.0)
+      jekyll (= 3.7.3)
+      jekyll-avatar (= 0.5.0)
+      jekyll-coffeescript (= 1.1.1)
+      jekyll-commonmark-ghpages (= 0.1.5)
+      jekyll-default-layout (= 0.1.4)
+      jekyll-feed (= 0.9.3)
+      jekyll-gist (= 1.5.0)
+      jekyll-github-metadata (= 2.9.4)
+      jekyll-mentions (= 1.3.0)
+      jekyll-optional-front-matter (= 0.3.0)
       jekyll-paginate (= 1.1.0)
-      jekyll-redirect-from (= 0.9.1)
-      jekyll-sass-converter (= 1.3.0)
-      jekyll-seo-tag (= 0.1.4)
-      jekyll-sitemap (= 0.9.0)
-      jekyll-textile-converter (= 0.1.0)
-      jemoji (= 0.5.1)
-      kramdown (= 1.9.0)
-      liquid (= 3.0.6)
+      jekyll-readme-index (= 0.2.0)
+      jekyll-redirect-from (= 0.13.0)
+      jekyll-relative-links (= 0.5.3)
+      jekyll-remote-theme (= 0.2.3)
+      jekyll-sass-converter (= 1.5.2)
+      jekyll-seo-tag (= 2.4.0)
+      jekyll-sitemap (= 1.2.0)
+      jekyll-swiss (= 0.4.0)
+      jekyll-theme-architect (= 0.1.1)
+      jekyll-theme-cayman (= 0.1.1)
+      jekyll-theme-dinky (= 0.1.1)
+      jekyll-theme-hacker (= 0.1.1)
+      jekyll-theme-leap-day (= 0.1.1)
+      jekyll-theme-merlot (= 0.1.1)
+      jekyll-theme-midnight (= 0.1.1)
+      jekyll-theme-minimal (= 0.1.1)
+      jekyll-theme-modernist (= 0.1.1)
+      jekyll-theme-primer (= 0.5.3)
+      jekyll-theme-slate (= 0.1.1)
+      jekyll-theme-tactile (= 0.1.1)
+      jekyll-theme-time-machine (= 0.1.1)
+      jekyll-titles-from-headings (= 0.5.1)
+      jemoji (= 0.9.0)
+      kramdown (= 1.16.2)
+      liquid (= 4.0.0)
+      listen (= 3.1.5)
       mercenary (~> 0.3)
-      rdiscount (= 2.1.8)
-      redcarpet (= 3.3.3)
-      rouge (= 1.10.1)
+      minima (= 2.4.1)
+      nokogiri (>= 1.8.1, < 2.0)
+      rouge (= 2.2.1)
       terminal-table (~> 1.4)
-    github-pages-health-check (0.6.0)
+    github-pages-health-check (1.4.0)
       addressable (~> 2.3)
       net-dns (~> 0.8)
-      public_suffix (~> 1.4)
-      typhoeus (~> 0.7)
-    html-pipeline (2.3.0)
-      activesupport (>= 2, < 5)
+      octokit (~> 4.0)
+      public_suffix (~> 2.0)
+      typhoeus (~> 1.3)
+    html-pipeline (2.7.1)
+      activesupport (>= 2)
       nokogiri (>= 1.4)
-    i18n (0.7.0)
-    jekyll (3.0.2)
-      colorator (~> 0.1)
+    http_parser.rb (0.6.0)
+    i18n (0.9.5)
+      concurrent-ruby (~> 1.0)
+    jekyll (3.7.3)
```

Comment:
looks like we're upgrading to jekyll 3.7.3. we'll have to make sure this works on the live server when we deploy